### ISSUE TITLE
Suggestion, sort of...

### ISSUE THREAD

**Issue by ergonomicus** [mateusz/silverstripe-frontend#19]
There are a lot of readme documents around that I find hard to follow and confusing. This might be one of those...

Can someone, please, explain me what "tl;dr" stands for?

Where should I put code for step 1?

Is it possible to add this functionality through CMS, since thsis is called widget?

It would be much easier if some links were provided...

**Comment by mateusz** [comment]
tl;dr is "too long, didn't read", a cusomary internet way of saying "this is a summary" ;-)

Step 1 goes into your DataObject, Controller, or wherever you are generating the item list from. There is more information in the [how-to](https://github.com/mateusz/silverstripe-frontend/blob/master/docs/sspagination-how-to.md).

This isn't really intended as a CMS, user-servicable module, although you could wrap it with code that does this.
